aip/cloud/new.md

```diff
@@ -0,0 +1,135 @@
+directly on the resource they intend it to apply to, and all children of the
+resource inherit the policy.
+
+A _shared_ policy is a first-class resource (with its own identifier) and can
+even have policies applied to it (like an authorization policy). Unlike other
+resources, a shared policy can be _bound_ to one or more resources, and applies
+to those resources and any descendants. An example of a shared policy is
+billing accounts, which are associated with resources that bill to them.
+
+## Guidance
+
+### Standard Methods
+
+For most policy types, the policy name **should** be incorporated into the
+method name. For example, `GetBillingPolicy` for a Billing policy. The
+descriptions below omit the specific policy type for brevity.
+
+#### Dedicated Policies
+
+If a resource can only have one of your policy type bound to it, and there does
+not need to be a separation of authorization privilege between the resource and
+the policy, then you **may** use a dedicated policy.
+
+Dedicated polices are the simpler of the two to implement, with a smaller API
+scope. The authorization policy applied to the resource applies to the policy
+as well. This can be problematic if users have permissions to modify
+permissions on the resource, yet aren't intended to be able to modify the
+policy.
+
+- **SetPolicy** - Sets a dedicated policy for a resource
+- **GetPolicy** - Retrieves the dedicated policy for a resource
+- **DeletePolicy** - Deletes the dedicated policy for a resource
+- **LookupEffectivePolicy** - Retrieves the effective policy for a resource,
+  which is the combination of policies on the resource and any ancestors
+
+#### Shared Policies
+
+If you want to bind a single policy to multiple resources, or want to apply a
+policy to your policy (eg., an authorization policy), then you **should** use a
+shared policy.
+
+- **CreatePolicy** - Creates a shared policy resource
+- **GetPolicy** - Retrieves a shared policy resource
+- **UpdatePolicy** - Updates a shared policy resource
+- **DeletePolicy** - Deletes a shared policy resource
+- **ListPolicies** - Lists shared policy resources (typically for a given
+  `parent`)
+- **LookupEffectivePolicy** - Retrieves the effective policy for a given
+  `resource`, which is the combination of the polices bound to the resource and
+  any ancestors
+- **CreateBinding** - Creates a binding between a shared policy resource and a
+  specified resource
+- **GetBinding** - Retrieves a binding between a shared policy resource and a
+  resource
+- **DeleteBinding** - Deletes a binding between a shared policy resource and a
+  resource
+- **ListBindings** - Lists bindings between shared policy resources and other
+  resources. This method **must** accept either a reference to the shared
+  policy resource or the target resource (typically via the `filter` field),
+  and list all appropriate bindings.
+
+#### Permissions
+
+Each policy method **should** have a separate permission, per standard
```

Comment:
I may have missed the part of this document (or another AIP) that "per standard practice" points back to?